Demos/Ethernet/EthernetDemo.cpp

```diff
@@ -29,10 +29,10 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE. */
 #include <vector>
 
 #ifdef SILKIT_HOURGLASS
-#include "silkit/hourglass/SilKit.hpp"
-#include "silkit/hourglass/config/IParticipantConfiguration.hpp"
+#    include "silkit/hourglass/SilKit.hpp"
```

Comment:
This looks funky 🤔

Reply:
Maybe BeforeHash for 
https://clang.llvm.org/docs/ClangFormatStyleOptions.html#indentppdirectives
Would be a better Option?

Reply:
Looks both funny, I turned preprocessor indent off.